### PR TITLE
Feature/log

### PR DESCRIPTION
**Pull request by ginesguiropa**
Log using OSLog

### CODE REVIEW

File: Source/Helpers/Log.swift [hunk 1]
```diff
@@ -42,3 +42,88 @@ public func Log (_ message: String,
...
+    case OSLogType.info:
+      return "ℹ️ℹ️ℹ️(info)"
+    case OSLogType.debug:
+      return "🔹🔹🔹(debug)"
+    case OSLogType.error:
+      return "‼️‼️‼️(error)"
+    case OSLogType.fault:
+      return "💣💣💣(fault)"
+    default:
+      return "⬜️⬜️⬜️(default)"
```
Comment (WedgeSparda) [inline Source/Helpers/Log.swift:112]: I see three gray squares. Is that correct?

File: Square1ToolsApp/Square1ToolsApp/Logs/LogsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import Square1Tools
+
+class LogsViewController: UIViewController {
+  
+  // MARK: - Actions
+  @IBAction func printLogs(_ sender: UIButton) {
+    Log("Default message", log: Logs.test)
+    Log("Debug message", log: Logs.test, type: .debug)
+    Log("Info message", log: Logs.test, type: .info)
+    Log("Ohter default message", log: Logs.test, type: .default)
```
Comment (WedgeSparda) [inline Square1ToolsApp/Square1ToolsApp/Logs/LogsViewController.swift]: Typo. Should be "Other"

File: Source/Helpers/Log.swift [hunk 3]
```diff
@@ -42,3 +42,88 @@ public func Log (_ message: String,
...
+/// - Parameters:
+///   - message: Object to be printed. Can be an expression.
+///   - log: OSLog object for this log.
+///   - type: OSLogType for this log.
+///   - logAccess: the level of privacity of the log. By default is public.
+///   - function: Name of the function calling this function. By default, same name as function where this is called.
+///   - line: Line number where this function is called. By default, line number within the file where the call is made.
+@available(OSX 10.12, iOS 10.0, watchOS 3.0, tvOS 10.0, *)
+public func Log(_ message: String,
+                log: OSLog,
```
Comment (WedgeSparda) [inline Source/Helpers/Log.swift]: Can we set a default value for log? So we can write a quick log with Log("Hello")

File: Source/Helpers/Log.swift (hunk 1 above)
Comment (ginesguiropa) [reply Source/Helpers/Log.swift:112]: Yes

Comment (WedgeSparda) [reply Source/Helpers/Log.swift:112]: Ok

File: Source/Helpers/Log.swift (hunk 3 above)
Comment (ginesguiropa) [reply Source/Helpers/Log.swift]: The old Log method has this method and we have a conflict with that

**Review by WedgeSparda — APPROVED** [review]

**Pull request merged**